middleware/http/oauth2/oauth2_middleware.go

```diff
@@ -81,6 +82,9 @@ func (m *Middleware) GetHandler(metadata middleware.Metadata) (func(h fasthttp.R
 			} else {
 				authState := session.GetString(savedState)
 				redirectURL := session.GetString(redirectPath)
+				if meta.ForceHTTPS == "true" {
```

Comment:
I would change ForceHTTP to lowercase first and compare with `"true"`

Reply:
Done.